src/resources/Clusters/Cluster.ts

```diff
@@ -0,0 +1,27 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {ClusterAgentModel, ClusterModel, ClusterStatusModel} from './ClusterInterfaces';
+
+export default class Cluster extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/clusters`;
+
+    list() {
+        return this.api.get<ClusterStatusModel[]>(Cluster.baseUrl);
+    }
+
+    listAgents(id: string) {
+        return this.api.get<ClusterAgentModel[]>(`${Cluster.baseUrl}/${id}/agents`);
+    }
+
+    live() {
+        return this.api.get<ClusterModel>(`${Cluster.baseUrl}/live`);
```

Comment:
This is the wrong type
![image](https://user-images.githubusercontent.com/35579930/67129862-b93ec880-f1cd-11e9-8edd-f36a7b5c3127.png)

Reply:
I named it differently but it's the good type